WireUI/Sources/WireIndividualToTeamMigrationUI/Views/TeamNameView.swift

```diff
@@ -35,14 +35,19 @@ struct TeamNameView: View {
                 .wireTextStyle(.body1)
             Spacer()
                 .frame(height: 24)
-            Text(String.localized(key: "individualToTeam.teamName.field.title", bundle: .module))
-                .wireTextStyle(.h4)
+            HStack(spacing: 4) {
+                Text(String.localized(key: "individualToTeam.teamName.field.title", bundle: .module)) +
+                    Text(verbatim: " *")
+                    .foregroundColor(ColorTheme.Base.requiredField.color)
+            }
+            .wireTextStyle(.h4)
```

Comment:
The `HStack` has no effect here, in my opinion, since two concatenated `Text` expressions will behave like a single content item within the `HStack`.